src/lio/bin/lio_mv.c

```diff
@@ -254,6 +261,7 @@ int main(int argc, char **argv)
             log_printf(15, "55555555555555555\n");
```

Comment:
I only commented because it was a +line on the diff
>
>

-- 
It's dark in this basement.